utilities/ipplan-import.py

```diff
@@ -0,0 +1,181 @@
+"""
+NIPAP import script for IPplan data.
+
+This script uses the standard export files base.txt and ipaddr.txt from Ipplan.
+"""
+import configparser
+import os
+# use local pynipap, useful if we are developing
+import sys
+sys.path.append('../pynipap')
+import pynipap
+import csv
+import ipaddress
+
+
+def add_ip_to_net(networks, host):
+    """
+    Add hosts from IPPlan to networks object.
+    """
+    for network in networks:
+        if host['ipaddr'] in network['network']:
+            network['hosts'].append(host)
+            return
+
+
+def get_networks(base_file, ipaddr_file):
+    """
+    Gather network and host information from IPPlan export files.
+    """
+    networks = []
+
+    base = open(base_file, 'r')
+
+    csv_reader = csv.reader(base.readlines(), delimiter='\t')
+    for row in csv_reader:
+
+        # Rows with less than 3 entries is most likely a newline.
+        if len(row) < 3:
```

Comment:
"likely"? :P 

how many types of lines are there? If you are looking for a newline I think it should be looked for more explicitly. If you want to also check that we have a minimum of X rows then do that but since it's really an unhandled case I think you should be explicit and throw an error rather than silently ignoring it.

Something like

```python
if len(row) == 1 and row[0] == '\n': # ignore newlines
    continue
if len(row) < 3: # not enough data, weird line -> throw error
    raise ValueError("Unhandled line...")
```